ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1064,7 +1064,7 @@ func getCreator(instance rayv1.RayCluster) string {
 	if instance.Labels == nil {
 		return ""
 	}
-	creatorName, exist := instance.Labels[utils.KubernetesCreatedByLabelKey]
+	creatorName, exist := instance.Labels[utils.RayOriginatedFromCRDLabelKey]
```

Comment:
The value is used to determine whether to inject certain environment variables for Ray Serve ([code](https://github.com/ray-project/kuberay/blob/f05fa2e28521b9b4024c81e3797a1544b9e1f025/ray-operator/controllers/ray/common/pod.go#L592-L610)). However, before #1830, the definition of `KubernetesCreatedByLabelKey` is pretty messy. In most cases, the value of the label is `kuberay-operator`, but RayService controller sets the label to `rayservice` until [#1830 deletes the line](https://github.com/ray-project/kuberay/pull/1830/files#diff-d3071444737c93f4b2e57426aba5e607f1063dc12f0b30e9d0595ce65b8620b8L675).